compiler/rustc_resolve/src/late/diagnostics.rs

```diff
@@ -3337,6 +3337,7 @@ impl<'ast, 'ra, 'tcx> LateResolutionVisitor<'_, 'ast, 'ra, 'tcx> {
                             .sess
                             .is_nightly_build()
                             .then_some(errors::ParamInNonTrivialAnonConstHelp),
+                        place: AnonConstKind::InlineConst,
```

Comment:
There is no context here so I am just defaulting to InlineConst. I need to fix this